packages/web-components/src/components/ic-radio-group/ic-radio-group.tsx

```diff
@@ -28,49 +28,49 @@ export class RadioGroup {
   @Element() host: HTMLIcRadioGroupElement;
 
   /**
-   * Provide a label for the input.
+   * The label for the input to be displayed.
```

Comment:
```suggestion
   * The label for the radio group to be displayed.
```
As we've replaced other occurrences of "input" should we here too?